cdc/sink/codec/avro.go

```diff
@@ -542,13 +542,31 @@ func newAvroEventBatchEncoderBuilder(credential *security.Credential, opts map[s
 		return nil, cerror.ErrPrepareAvroFailed.GenWithStack(`Avro protocol requires parameter "registry"`)
 	}
 
+	topic, ok := opts["topic"]
+	if !ok {
+		topic = ""
+	}
+
+	nameStrategy, ok := opts["nameStrategy"]
```

Comment:
Is this new option `nameStrategy` mandatory or optional. Is the default value of this new option `legacy`?

Reply:
The `nameStrategy` is optional and the default is `legacy`. Only when set explicitly it will use the new behavior.